contrib/containerized-pbench/pbench

```diff
@@ -0,0 +1,70 @@
+#! /bin/bash
```

Comment:
Since we can't comment on a new files name ... can we rename this to something like `pbench-container-exec`?  I'd rather not pollute the `pbench` name that we want to move to for the agent commands going forward.

Reply:
Actually, I deliberately chose to use the `pbench` name in an attempt to model the going-forward Agent command format.  I was hoping that we could end up with a command like
```
$ pbench results push
```
where the initial `pbench` would run the proposed script to invoke the `pbench` Agent command inside the container, and the `results` and `push` arguments would be passed through to it, so it would actually end up running the `pbench results push` command inside the container.

Obviously this isn't practical today, since we don't have a standalone `pbench` command, yet, but this could form the basis for it.

Reply:
The going forward agent command is not going to containers for its operation.

Reply:
The going-forward Agent command would only be available if the environment has the Pbench Agent installed.

However, instead of installing the Pbench Agent, the user could use a wrapper around a Podman `run` command which invokes a container whose entrypoint is the going-forward Agent `pbench` command.

If the wrapper is invoked via a script or alias which is `pbench`, then there will be basically no difference in the user experience between running an installed Agent vs. running a containerized one.  E.g.,
```
$ pbench results push
```
could invoke the natively-installed Pbench Agent or it could invoke a containerized Agent (depending on what the `pbench` command resolves to), and either way it would have the same effect (assuming that the respective versions of the Agent were congruent).

So, no, we don't _have_ to use a containerized Pbench Agent going forward, but we _could_, and it might prove to be convenient for users.